jmeter-java-dsl-csv-random/src/main/java/us/abstracta/jmeter/javadsl/csvrandom/RandomCsvDataSetConfig.java

```diff
@@ -0,0 +1,165 @@
+package us.abstracta.jmeter.javadsl.csvrandom;
+
+import com.blazemeter.jmeter.RandomCSVDataSetConfig;
+import com.blazemeter.jmeter.RandomCSVDataSetConfigGui;
+import org.apache.jmeter.testelement.TestElement;
+import us.abstracta.jmeter.javadsl.core.testelements.BaseTestElement;
+import us.abstracta.jmeter.javadsl.core.testelements.MultiLevelTestElement;
+
+/**
+ * The Random CSV Data Set Config plugin provides the capability to parameterize your Test Plan
+ * from a CSV file in random order of records
```

Comment:
here checkstyle will complain about no dot at end of first sentence of javadoc

Reply:
I see you marked this as resolved. Isn't checkstyle complaining about the dot missing in the end of this sentence.